### PR TITLE
PR template Contribution Guidelines link doesn't work

### LINKED ISSUE #134: PR template Contribution Guidelines link doesn't work
Just helps new starters to get started with how to contribute when opening a PR. I am happy to attempt this.

Screenshot of Contribution Guidelines hyperlink:
image

### Checklist
- [x] If possible, I've reproduced the issue using the `main` branch of this package
- [x] I've searched for [existing GitHub issues](https://github.com/apple/swift-algorithms/issues)

### Steps to Reproduce
The link should be (I think): https://github.com/apple/swift-algorithms/blob/main/CONTRIBUTING.md
Actual: https://github.com/apple/swift-algorithms/CONTRIBUTING.md <-- This is broken for #133

### Expected behavior
Open Markdown file for Contribution Guidelines.

### Actual behavior
image

### PR DESCRIPTION
**Pull request by LemonSpike**
Fixes #134.

### Checklist
- [x] I've added at least one test that validates that my change is working, if appropriate (this PR itself!! 😆)
- [x] I've followed the code style of the rest of the project
- [x] I've read the [Contribution Guidelines](../blob/main/CONTRIBUTING.md)
- [x] I've updated the documentation if necessary

### CODE REVIEW

**Comment by LemonSpike** [comment]
@swift-ci please test

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
     If this pull request adds new API, please add '?template=new.md'
     to the URL to switch to the appropriate template.
-
+
```
Comment (xwu) [inline .github/PULL_REQUEST_TEMPLATE.md:6]:
```suggestion

```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -3,7 +3,6 @@
 
     If this pull request adds new API, please add '?template=new.md'
     to the URL to switch to the appropriate template.
-
```
Comment (xwu) [inline .github/PULL_REQUEST_TEMPLATE.md:6]: Hmm, that shouldn't have happened; please restore the original here.

**Comment by timvermeulen** [comment]
@swift-ci please test

**Comment by timvermeulen** [comment]
Thanks for fixing this!

**Pull request merged**